src/llfs/ioring_log_device2.test.cpp

```diff
@@ -588,6 +589,11 @@ TEST(IoringLogDevice2Test, Benchmark)
     // Run the passed workload.
     //
     workload_fn(log_device);
+
+    const double write_amplification =
+        double(log_device.metrics().bytes_written) / double(log_device.metrics().bytes_flushed);
+
+    LLFS_LOG_INFO() << log_device.metrics() << BATT_INSPECT(write_amplification);
```

Comment:
I didn't see any metric counting "fully written flush_tail page" cases. It will good to have a metric around that which a test could use to make sure this scenario is covered through at least one test.

Reply:
In general it will be good to verify some collected metric values to make sure tests are covering intended scenarios. For example, if burst_mode_checked is non-zero then burst_mode_applied should be non-zero for a test to pass.

Reply:
I'm not sure what you mean by "fully written flush_tail" case, but I think I still see the point.  There are several control paths that don't have metric counters and it would be good (for testing and for diagnostics possibly too) to track how often they occur.

Reply:
This is the scenario when last page is fully filled (in order words commit aligning with the page boundary).

Reply:
Added metrics and verified them in the test code.